### DISCUSSION TITLE
CANEVA PRO

### DISCUSSION THREAD

**Discussion by mariefarouze** [community/community#45817] [GitHub Education] [answered]
Hello,
I need help please ... 
I created my account GitHub education but I can't find access to Caneva Pro.
In the Student Developer Pack offers, the site does not appear.
how to do ?

**Comment by juanpflores** [comment, marked as answer]
Hi @mariefarouze  👋 ! Thanks for the great question about Global Campus Benefits.

We no longer provide the Canva benefit as part of the Student Developer Pack. You can check out all the Student Developer Pack's latest offers and partners available [here](https://education.github.com/pack?sort=az#offers). For students who have already claimed for your 12 month benefit, you will continue to have access for the remainder of your 12 month offer period and can contact [Canva's support team](https://www.canva.com/help/get-in-touch/contact-support/) with any related questions.
